README.md

```diff
@@ -1,5 +1,35 @@
 # bus-sdk-python
 
-[![Build Status](https://alfred.elifesciences.org/buildStatus/icon?job=library-bus-sdk-python)](https://alfred.elifesciences.org/job/library-bus-sdk-python/) [![Build Status](https://travis-ci.org/elifesciences/bus-sdk-python.svg?branch=master)](https://travis-ci.org/elifesciences/bus-sdk-python) [![Coverage Status](https://coveralls.io/repos/github/elifesciences/bus-sdk-python/badge.svg?branch=add_travis_and_coveralls)](https://coveralls.io/github/elifesciences/bus-sdk-python?branch=add_travis_and_coveralls)
+[![Build Status](https://alfred.elifesciences.org/buildStatus/icon?job=library-bus-sdk-python)](https://alfred.elifesciences.org/job/library-bus-sdk-python/) [![Coverage Status](https://coveralls.io/repos/github/elifesciences/bus-sdk-python/badge.svg?branch=add_travis_and_coveralls)](https://coveralls.io/github/elifesciences/bus-sdk-python?branch=add_travis_and_coveralls)
 
-Python SDK for the eLife Sciences bus [in development]
+This library provides a Python SDK for the [eLife Sciences Bus](https://github.com/elifesciences/bus).
+    
+
+Dependencies
+------------
+
+* Python >=3.5
```

Comment:
Interesting, so it requires to upgrade a project to Python 3 before using it? It's fine as it keep up the pressure to do so.
Was it a particular dependency that spawned this or just the general ADR?

Reply:
Yes making the whole library backwards compatible did not seem like the right thing to do in regards to the Python 3 push we are doing. As older projects will already have something in place (to handle SQS/SNS etc) then this can just be a part of an upgrade process.